src/PostgREST/Plan.hs

```diff
@@ -153,18 +153,20 @@ dbActionPlan dbAct conf apiReq sCache = case dbAct of
 
 wrappedReadPlan :: QualifiedIdentifier -> AppConfig -> SchemaCache -> ApiRequest -> Bool -> Either Error CrudPlan
 wrappedReadPlan  identifier conf sCache apiRequest@ApiRequest{iPreferences=Preferences{..},..} headersOnly = do
-  rPlan <- readPlan identifier conf sCache apiRequest
-  (handler, mediaType)  <- mapLeft ApiRequestError $ negotiateContent conf apiRequest identifier iAcceptMediaType (dbMediaHandlers sCache) (hasDefaultSelect rPlan)
+  qi <- mapLeft ApiRequestError $ findTable identifier (dbTables sCache)
+  rPlan <- readPlan qi conf sCache apiRequest
```

Comment:
Similar to how to we `findProc` before building the call plan, I think this approach should suffice without needing a lot of refactoring for a quick fix.

Reply:
Good thinking! :100: Code looks good now.